src/StackExchange.Redis/RedisServer.cs

```diff
@@ -638,12 +646,14 @@ public void SlaveOf(EndPoint master, CommandFlags flags = CommandFlags.None)
             }
         }
 
-        public Task SlaveOfAsync(EndPoint master, CommandFlags flags = CommandFlags.None)
+        Task IServer.SlaveOfAsync(EndPoint master, CommandFlags flags) => ReplicaOfAsync(master, flags);
```

Comment:
This went private - public API break when we should obsolete?

Reply:
`internal sealed class RedisServer : RedisBase, IServer` - it is accessed via the `IServer` interface; all this did was make the internal-code-accessible version the current one, and make the legacy one explicit interface implementation